statistics/handle/statscache.go

```diff
@@ -15,15 +15,48 @@ package handle
 
 import (
 	"encoding/binary"
+	"errors"
 	"sync"
 
 	"github.com/pingcap/tidb/statistics"
 	"github.com/pingcap/tidb/util/kvcache"
 	"github.com/pingcap/tidb/util/memory"
 )
 
-// statsCache caches table statistics.
-type statsCache struct {
+// StatsCache a interface can LookUp Update
+type StatsCache interface {
+	Lookup(id int64) (*statistics.Table, bool)
+	Update(tables []*statistics.Table, deletedIDs []int64, newVersion uint64)
+	GetVersion() uint64
+	InitStatsCache(tables map[int64]*statistics.Table, version uint64)
+	GetAll() []*statistics.Table
+
+	// interface below are used only for test
```

Comment:
Capitalize the first character of the first comment word, and end the line with a `.`.